org.lflang/src/org/lflang/generator/GeneratorBase.xtend

```diff
@@ -1505,6 +1508,42 @@ abstract class GeneratorBase extends AbstractLFValidator implements TargetTypes
             return time.time.toString()
         }
     }
+    
+    
+    
+    /**
+     * Remove disconnected network ports in federates, if any.
```

Comment:
I think already mentioned this elsewhere in the PR, but I find the term "disconnected network port" confusing. I think it would be better to call it something like `removeFederateConnectionPorts` (because the interfederate connections are being substituted by network connections).

Reply:
Thanks. That's a good point. I changed the name and documentation.